plugins/occt/CMakeLists.txt

```diff
@@ -71,6 +71,15 @@ f3d_plugin_declare_reader(
   CUSTOM_CODE "${CMAKE_CURRENT_SOURCE_DIR}/brep.inl"
 )
 
+f3d_plugin_declare_reader(
+  NAME XBF
+  EXTENSIONS xbf
+  MIMETYPES application/vnd.xbf
+  VTK_READER vtkF3DOCCTReader
+  FORMAT_DESCRIPTION "Open CASCADE XBF"
+  CUSTOM_CODE "${CMAKE_CURRENT_SOURCE_DIR}/xbf.inl"
+)
```

Comment:
which version of OCCT did you test this locally with ?

We are currently stuck on 7.7  but testing with 7.8 would be nice.

Reply:
I've only got `7.6` on my system for now :/

Reply:
ok, lets keep this open until someone (can be me :) ) test on 7.8

Reply:
it works with 7.8